docs/architecture/adr-069-gov-improvements.md

```diff
@@ -0,0 +1,193 @@
+
+#### Optimistic Proposal
+
+An optimistic proposal is a proposal that automatically passes unless a threshold a NO votes is reached.
+
+Voter can only vote NO on the proposal. If the NO threshold is reached, the optimistic proposal is converted to a normal proposal.
+
+Two governance parameters will be in added [`v1.Params`][5] to support optimistic proposals:
+
+```protobuf
+// optimistic_authorized_addreses is an optional governance parameter that limits the authorized accounts than can submit optimisitc proposals
+repeated string optimistic_authorized_addreses = 17 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+
+// Optimistic rejected threshold defines at which percentage of NO votes, the optimistic proposal should fail and be converted to a normal proposal.
+string optimistic_rejected_threshold = 18 [(cosmos_proto.scalar) = "cosmos.Dec"];
+```
+
+#### Multiple Choice Proposal
+
+A multiple choice proposal is a proposal where the voting options can be defined by the proposer.
+
+The number of voting option will be limited to a maximum of 4. A new vote option `SPAM` will be added and distinguished of those voting options. 
+
+Multiple choice proposals, contrary to any other proposal type, cannot have messages to execute. They are only text proposals.
+
+Submitting a new multiple choice proposal will use a different message than the [`v1.MsgSubmitProposal`][5]. This is done in order to simplify the proposal submittion and allow defining the voting options directly.
+
+
+```protobuf
+message MsgSubmitMultipleChoiceProposal {
+  repeated cosmos.base.v1beta1.Coin initial_deposit = 1
+  string proposer = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  string metadata = 3;
+  string title = 4;
+  string summary = 5;
+  VoteOption option_one = 6;
+  VoteOption option_two = 7;
+  VoteOption option_three = 8;
+  VoteOption option_four = 9;
+}
+```
+
+Voters can only vote on the defined options in the proposal.
+
+To maintain compatibility with the existing endpoints, the voting options will not be stored in the proposal itself. A multiple choice proposal will be stored as a [`v1.Proposal`][5]. A query will be available for multiple choice proposal types to get the voting options.
+
+Multiple choice proposals do not need additional governance parameters.
+
+### Votes
+
+As mentioned above [multiple choice proposal](#multiple-choice-proposal) will introduce an additional vote option: `SPAM`.
+
+This vote option will be supported by all proposal types.
+Voting `SPAM` will automatically burn the proposal deposit and mark the proposal as failed.
+
+`SPAM` differs from the `No with Veto` vote as its threshold is dynamic.
+A proposal is marked as `SPAM` when the total of weighted votes for all options is lower than the amount of weighted vote on `SPAM`
+(`spam` > `option_one + option_two + option_three + option_four` = proposal marked as spam).
```

Comment:
What happens if the first vote on the proposal is a SPAM vote? I suppose it won't be marked as SPAM right away, will it?

Reply:
Currently clients are using the hack of not displaying proposals with a high number of NWV. This allows them to only filter by spam proposal.
A proposal will be marked as spam when tallying at the end of the voting period, so the fact the first voter votes SPAM does not matter.

Reply:
OK, I thought the SPAM mark was immediate and interrupted the voting period. I think what confused me is this sentence:

> Voting `SPAM` will automatically burn the proposal deposit and mark the proposal as failed.

I interpret *automatically* as *immediately*. Thanks for the correction.